liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -467,6 +468,7 @@ public int getPrecedence() {
         } else {
             Scope.getCurrentScope().getLog(getClass()).fine("Cannot find local defaultsFile " + defaultsFile.getAbsolutePath());
         }
+        liquibaseConfiguration.registerProvider(new InteractivePromptingValueProvider());
```

Comment:
I like the idea of plugging interactions into the ConfigurationValueProvider system.

Reply:
This solves a lot of problems, so I'm glad you're on board.